lavalink/WebSocket.py

```diff
@@ -136,7 +136,7 @@ def connected(self):
                 await self._lavalink.update_state(data)
             elif op == 'stats':
                 self._lavalink.stats._update(data)
-                await self._lavalink.dispatch_event(RawStatsUpdateEvent(data))
+                await self._lavalink.dispatch_event(StatsUpdateEvent(data))

```

Comment:
Would dispatching `self._lavalink.stats` to prevent recreation be better?

Reply:
Actually yes, I'd remove some unnecessary processing.